strabo/strabo/views/public.py

```diff
@@ -7,24 +7,12 @@
 get_geojson, search, gallery_search
 from strabo.geojson import make_featureCollection
 from strabo.utils import prettify_columns, get_raw_column
+import copy
 
 @app.route("/map")
 def map():
   template = os.path.join("public/", app.config['MAP_TEMPLATE'])
```

Comment:
Flasks integration with `jinja2` will auto(matically/magically) find templates by name, if that's of interest to y'all. (You'll need to put your templates in a specific location, but that's not difficult). I don't think this way is wrong -- just depends on how you want it to work.